cmd/securetrigger/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/hex"
+	"log"
+	"os"
+)
+
+func main() {
```

Comment:
While the logic here is fairly simple, it would be nice to have some unit tests for this :angel: